eogrow/core/area/base.py

```diff
@@ -121,20 +138,12 @@ def get_grid_cache_filename(self) -> str:
 
     def get_patch_list(self) -> PatchList:
```

Comment:
Would this function also benefit from having the flag?

I guess it's better to keep it simple and have a 1 to 1 relation to what is used. All patches can be obtained manually in case they are needed.

Reply:
I think even the grid doesnt necessarily need to have the flag. Sometimes it's better to not give the user too many options to think about. The patch list should be filtered, and the user should think real hard on why someone would specify filtering, but then have it ignored.

If filtering is not allowed for a pipeline, one can set the type in the schema to `Literal[None]` similar to what is sometimes done for `skip_existing`.

It is also easier to add functionality that to take it away, so I'd prefer to add it later if need shows.